packages/@o3r/workspace/schematics/sdk/index.ts

```diff
@@ -44,7 +43,7 @@ export function generateSdk(options: NgGenerateSdkSchema): Rule {
     const yarnrcBeforeSdkGeneration = tree.exists(yarnrcPath) ? tree.readText(yarnrcPath) : '';
 
     return chain([
-      externalSchematic<NgGenerateTypescriptSDKShellSchematicsSchema>('@ama-sdk/schematics', 'typescript-shell', {
```

Comment:
Why is it removed?
If the remove of the type is justified, is it not possible to remove the dep to `@ama-sdk/schematics` as well?